LiskHQ/lisk-hub
# Add language select to settings - Closes #2348

### What issue have I solved?

#2348

### How have I implemented/fixed it?

- [x] Add Language select to Settings
- [x] Enable the logic to sync lang between React and Electron app
- [x] Enable saving language in dev mode https://github.com/LiskHQ/lisk-hub/pull/2349/commits/d1c5d5a5187652ec85818ef2b0f178f95323072c
- [x] Hide it from UI under `feature-flag-language` localStorage value by default because the German strings are not added yet (goal of another ticket, at which point the feature flag can be reverted by `git revert 99e12e0a82220f1b77d65ab11ee2cbc010a5c398`).

### How has this been tested?

It's needed to set a `feature-flag-language` into localStorage to see the option in settings. 
```js
localStorage.setItem('feature-flag-language', true);
```

- Go to Settings
- Change language
- See that some strings are already in German (e.g. `Settings` -> `Einstellungen`)
### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
